jscomp/bin/bsdep.ml

```diff
@@ -30431,7 +30431,7 @@ let handle_raw ?(check_js_regex = false) loc payload =
     | Not_String_Lteral ->
       Location.raise_errorf ~loc
         "bs.raw can only be applied to a string"
-    | Not_String_Lteral ->
+    | Ast_payload.JS_Regex_Check_Failed ->
```

Comment:
Ah, I normally do `make forcesnapshot...` and the do `make bsc.exe` to build the compiler, not really sure how this is not detected.